dbt_copilot_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -432,6 +456,8 @@ export class TransformedStack extends cdk.Stack {
         this.codestarConnection = {arn: codestarConnectionArn, id: codestarConnectionId};
     }
 
+
+
```

Comment:
Some whitespace snuck past black somehow?

Reply:
@antroy-madetech urgh. well, fixed. going to merge it in now and push the same fix to demodjango-deploy